.github/workflows/product_metadata_checks.yml

```diff
@@ -1,4 +1,33 @@
-name: Product_metadata_checks
+name: 🧪 Validate Product Metadata
+run-name: 🧪 Validate Product Metadata (${{ github.event_name }})
 
 on:
+  pull_request:
```

Comment:
I was going to write - maybe make this part of the pr test action?

But thinking about it... maybe legibility of having some separate actions is actually a good thing, rather than continuing to add to the bloat in our pr test job.

One downside of this is that it won't make use of the logic to create dev images for running tests in. However this specific test doesn't seem like that would present an issue.

Thoughts?

Reply:
yeah, I think you're right. Can't recall why I didn't just add it there. Will change.

Reply:
That'd be `test.yml` for the path filter logic and test_helper then for the actual tests - top level `test.yml` takes care of the docker image logic and calls test_helper with image tag and path_filters output as args. These were separated to make it easier to call the tests within `test_helper` from the nightly_qa job

Individual jobs in test_helper look like this
```yml  
template-db:
    if: (inputs.path_filter == '') || contains(inputs.path_filter, 'template')
```
With the first clause in the `if` there being in the case of nightly_qa (where path_filters job wasn't run, we just run all tests)

Reply:
So you could also just leave this as its own yml file and just call it from one of the other actions